src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -83,13 +95,15 @@ public void setReceiver(String username) {
     super.setReceiver(username);
     privateChatTabRoot.setId(username);
     privateChatTabRoot.setText(username);
-
+    playerService.getPlayerByNameIfOnline(username).ifPresent(player ->
+        JavaFxUtil.addAndTriggerListener(player.avatarProperty(), new WeakChangeListener<>(avatarPropertyListener)));
     ChatChannelUser chatUser = chatService.getOrCreateChatUser(username, username, false);
     privatePlayerInfoController.setChatUser(chatUser);
   }
 
   public void initialize() {
     super.initialize();
+    JavaFxUtil.bindManagedToVisible(avatarImageView);
```

Comment:
```suggestion
    JavaFxUtil.bindManagedToVisible(avatarImageView);
    avatarImageView.visibleProperty().bind(avatarImageView.imageProperty().isNotNull());
```

This will remove the need for the explicit setVisible call below